model/src/commonMain/kotlin/runs/evaluator/RuleViolation.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.model.runs.evaluator
+
+import org.eclipse.apoapsis.ortserver.model.Severity
+
+data class RuleViolation(
```

Comment:
Question: There is already the `OrtRuleViolation` class under `model`. It contains the same properties and an identifier. Is there a reason why this class cannot be used here?

Reply:
I already spotted this, and it's removed and replaced with `OrtRuleViolation` class.